cmd/capibmadm/printer/printer.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
any specific reason for moving from utils -> printer?

Reply:
Facing circular import issue after using printer type in options pkg. Also we are introducing validation as well in printer, so its better to keep it in separate pkg.